apps/condo/domains/ticket/utils/serverSchema/ticketBlankExport.js

```diff
@@ -0,0 +1,593 @@
+        pdfDoc.on('end', () => {
+            resolve(Buffer.concat(chunks))
+        })
+        pdfDoc.end()
+    } catch (e) {
+        reject(e)
+    }
+})
+
+const createPdfForTicketBlanks = async ({ tickets, blankOptions, locale }) => {
+    const ticketPdfBuffers = []
+
+    for (const ticket of tickets) {
+        const comments = await getComments({ ticket, blankOptions })
+        const ticketBlankData = convertTicketToTicketBlank({ ticket, comments, blankOptions, locale })
+        const translations = getTranslations({ locale, ticketBlankData })
+        const replaces = {
+            blank: ticketBlankData,
+            i18n: translations,
+        }
+        const pdf = generatePdf(replaces)
+        const buffer = await pdfToBuffer(pdf)
+        ticketPdfBuffers.push(buffer)
+    }
+
+    const mergedPdf = await mergePdfBuffers(ticketPdfBuffers)
+
+    const stream = Readable.from(mergedPdf)
+
+    return { stream }
+}
+
+const getTicketBlankOptions = (options) => {
+    const commentIds = get(options, 'commentIds')
+    const haveAllComments = get(options, 'haveAllComments')
+    const haveListCompletedWorks = get(options, 'haveListCompletedWorks')
+    const haveConsumedMaterials = get(options, 'haveConsumedMaterials')
+    const haveTotalCostWork = get(options, 'haveTotalCostWork')
+
+    return {
+        commentIds: isArray(commentIds) ? commentIds : [],
+        haveAllComments: isBoolean(haveAllComments) ? haveAllComments : false,
+        haveListCompletedWorks: isBoolean(haveListCompletedWorks) ? haveListCompletedWorks : false,
+        haveConsumedMaterials: isBoolean(haveConsumedMaterials) ? haveConsumedMaterials : false,
+        haveTotalCostWork: isBoolean(haveTotalCostWork) ? haveTotalCostWork : false,
+    }
+}
+
+const exportTicketBlanksToPdf = async ({ context, task, baseAttrs, where, sortBy }) => {
+    const { id: taskId, options, locale } = task
+
+    const blankOptions = getTicketBlankOptions(options)
+
+    const tickets = await loadTicketsForPdfExport({ where, sortBy, limit: MAX_TICKET_BLANKS_EXPORT })
+
+    const { stream } = await createPdfForTicketBlanks({ tickets, blankOptions, locale })
+
+    const fileInput = {
+        stream,
+        filename: `ticket_blanks_${dayjs().format('DD_MM_YYYY__HH_mm_ss')}.pdf`,
+        mimetype: PDF_FILE_META.mimetype,
+        encoding: PDF_FILE_META.encoding,
+        meta: {
+            listkey: 'TicketExportTask',
+            // Id of first record will be used by `OBSFilesMiddleware` to determine permission to access exported file
+            // NOTE: Permissions check on access to exported file will be replaced to checking access on `ExportTicketsTask`
+            id: taskId,
+        },
+    }
+
+    const fileUploadInput = buildUploadInputFrom(fileInput)
+
+    await TicketExportTask.update(context, taskId, {
+        ...baseAttrs,
+        status: COMPLETED,
+        file: fileUploadInput,
+    })
+}
+
+module.exports = {
+    exportTicketBlanksToPdf,
```

Comment:
Could you please also rename the module to `exportTicketBlanksToPdf` (verb-first format).